client/webserver/locales/en-us.go

```diff
@@ -283,4 +283,11 @@ var EnUS = map[string]string{
 	"Disabled":                    "Disabled",
 	"txfee_not_available":         "Transaction fee currently unavailable",
 	"fiat_exchange_rate_msg":      "Sources may provide fiat exchange rates for different subsets of assets. You should select all sources that are acceptable to get fiat exchange rates for the most assets. Assets with fiat rates from multiple sources will use the average from all of those sources. Note: dcrdate provides fiat exchange rates for only BTC and DCR.",
+	"delete_archived_records":     "Delete Archived Records",
+	"date_time":                   "Date and Time",
+	"archived_date_time_tooltip":  "Archived orders and matches created before your specified and date and time will be deleted from the database. Note: If an invalid date and time is provided, all archived records till date will be deleted.",
+	"matches_file_path":           "Matches CSV File Path",
+	"matches_file_path_tooltip":   "Optional: Specify a file path to save deleted matches as a CSV file. File must not already exist.",
+	"orders_file_path":            "Orders CSV File Path",
```

Comment:
Because of how `(*Core).DeleteArchivedRecords` takes file paths and writes to the disk of the machine where dexc is running, I think we have to do it like this. But it's a notable difference from how the order data export works by serving the exported file over the web browser.

To make it work like that, we'd have to ask Core to write to a temp folder and then serve the written files somehow.  Not sure it's worth it though.  Most people will either be running dexc on the same machine as the browser, or have easy access to the machine.

Reply:
> takes file paths and writes to the disk of the machine where dexc is running, I think we have to do it like this.

Yeah, if a user provides just a filename, Core will write to `pwd` i.e the `/dexc` folder. I added a check for that in `core.go` `createFile(fileName string)`.

Reply:
I'm not commenting about the cwd issue raised in matrix, but the fact that this can be a different machine.

Reply:
> I'm not commenting about the cwd issue raised in matrix, but the fact that this can be a different machine.

Oh, okay. We could provide the `datadir` to Core so we can save files like these to `datadir` instead?